rcgen/src/key_pair.rs

```diff
@@ -1,30 +1,41 @@
 #[cfg(feature = "pem")]
 use pem::Pem;
+#[cfg(feature = "crypto")]
 use std::convert::TryFrom;
 use std::fmt;
 use yasna::DERWriter;
 
+#[cfg(any(feature = "crypto", feature = "pem"))]
 use crate::error::ExternalError;
+#[cfg(feature = "crypto")]
```

Comment:
it would be better to have one use block like `use crate::{...}` instead of copy-pasting this to all instances

Reply:
That's a good point too, I've now done this.